src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> lst = new ArrayList<Task>();
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hey! I'm Duke!");
+        System.out.println("What can I help you with?");
+        boolean done = false;
+        int tasks = 0;
+        while (done != true) {
+            String output = sc.nextLine();
+            if (validationInput(output)) {
+                if (output.equals("bye")) {
+                    System.out.println("Bye! See you around:)");
+                    done = true;
+                } else if (output.equals("list")) {
+                    int counter = 1;
+                    System.out.println("    ____________________________________________________________");
+                    for (int i = 0; i < lst.size(); i++) {
+                        if (!lst.get(i).getType().equals("todo")) {
+                            System.out.println("      " + counter + ".[" + initialsT(lst.get(i).getType()) + "]" +"[" + lst.get(i).getStatusIcon() + "] " +lst.get(i).getD() + "(" + breakDate(lst.get(i).getTime() +")") +")");
+                        } else {
+                            System.out.println("      " + counter + ".[" + initialsT(lst.get(i).getType()) + "]" +"[" + lst.get(i).getStatusIcon() + "] " +lst.get(i).getD() );
```

Comment:
style - line is too long, split it to adhere to style guide, also watch out for minor spacing issues '+ "["' instead of '+"["'.